src/components/ProjectListRow.js

```diff
@@ -53,8 +56,17 @@ class ProjectListRow extends React.Component {
 	resetLoader = () => {
 		this.props.showLoader(false);						
 	    this.setState({importText: AutographaStore.currentTrans["btn-import"], isImporting: false})
-    }
-  	importBook = (projectId) => {
+    };
+
+	importBook = (projectId) => {
+		if (this.props.paratextObj instanceof Wacs) {
+			return this.importBookWacs(projectId);
```

Comment:
This code path covers d43 too.  I pretty much used "WACS" in identifiers to mean the WACS/Door43/Gitea path.  

(BTW, also visible in this snippet is `paratextObj`, which is used for all sync providers, not just Paratext.  I limited my refactoring and renaming because I'm a guest in this codebase.  There is certainly opportunity for refactoring, but maybe that should go in another PR?)

Reply:
Ah.. Yes, this would need to refactored but we can include it in the next PR.